exchanges/okgroup/okgroup_types.go

```diff
@@ -666,7 +666,7 @@ type GetFuturesTransactionDetailsResponse struct {
 
 // GetFuturesContractInformationResponse individual contract details from  GetFuturesContractInformation
 type GetFuturesContractInformationResponse struct {
-	ContractVal     int64   `json:"contract_val,string"`
+	ContractValue   float64 `json:"contract_val,string"`
 	Delivery        string  `json:"delivery"`
```

Comment:
Out of scope; but there are a few missing fields: 
Alias string `json:"alias"`
Underlying string `json:"underlying"`
SettlementCurrency string `json:"settlement_currency"`
IsInverse bool `json:"is_inverse"`
ContractValueCurrency string `json:"contract_val_currency"`

Reply:
Missing fields addressed